qiskit/ignis/verification/quantum_volume/fitters.py

```diff
@@ -302,32 +308,72 @@ def plot_qv_data(self, ax=None, show_plt=True):
             plt.show()
 
     def qv_success(self):
-        """Return whether each depth was successful (>2/3 with confidence
-        greater than 97.5) and the confidence
+        """Return whether each depth was successful (> 2/3 with confidence
+        level > 0.977 corresponding to z_value = 2) and the confidence level
 
         Returns:
             list: List of lenth depth with eact element a 3 list with
             - success True/False
-            - confidence
+            - confidence level
         """
 
         success_list = []
+        confidence_level_threshold = self.calc_confidence_level(z_value=2)
 
         for depth_ind, _ in enumerate(self._depths):
             success_list.append([False, 0.0])
             hmean = self._ydata[0][depth_ind]
-            if hmean > 2/3:
-                cfd = 0.5 * (1 +
-                             math.erf((hmean - 2/3)
-                                      / (1e-10 +
-                                         self._ydata[1][depth_ind])/2**0.5))
-                success_list[-1][1] = cfd
+            sigma = self._ydata[1][depth_ind]
+            z_value = self.calc_z_value(hmean, sigma)
+            confidence_level = self.calc_confidence_level(z_value)
+            success_list[-1][1] = confidence_level
 
-                if cfd > 0.975:
-                    success_list[-1][0] = True
+            if (hmean > 2/3 and confidence_level > confidence_level_threshold):
+                success_list[-1][0] = True
 
         return success_list
 
+    def calc_z_value(self, mean, sigma):
+        """
+        Calculate z value using mean and sigma
+
+        Args:
+            mean (float): mean
+            sigma (float): standard deviation
+
+        Raises:
+            ValueError: If sigma is zero.
+
+        Returns:
+            float: z_value in standard normal distibution.
+        """
+
+        if sigma == 0:
+            raise ValueError('Standard deviation sigma should not be zero.')
+
```

Comment:
@dcmckayibm I added raising ValueError here. Do you think this is good?

Reply:
I would probably raise a warning that the sigma is zero and then use a small value of sigma (e.g. 1e-10) so that the code will still run.

Reply:
I realise that it's almost impossible to have sigma = 0 even if `ntrial = 1` because sigma is calculated using this code `self._ydata[1][depthidx] = (self._ydata[0][depthidx] *
                                        (1.0-self._ydata[0][depthidx])
                                        / self._ntrials)**0.5`, unless `self._ydata[0][depthidx]=0` (hmean) is 0. 

Anyway, I change ValueError to warning as you suggested.